phantom_token.c

```diff
@@ -488,202 +502,96 @@ static ngx_int_t handler(ngx_http_request_t *request)
     return NGX_AGAIN;
 }
 
-static ngx_int_t set_www_authenticate_header(ngx_http_request_t *request, phantom_token_configuration_t *module_location_config, char *error_code)
```

Comment:
I moved this lower down without changing it so that the code reads in a more logical sequence.